addons/website_blog/controllers/main.py

```diff
@@ -63,12 +64,43 @@ def blogs(self, page=1, **post):
             page=page,
             step=self._blog_post_per_page,
         )
+        date_begin, date_end = post.get('date_begin'), post.get('date_end')
+        if date_begin and date_end:
+            domain += [("post_date", ">=", date_begin), ("post_date", "<=", date_end)]
+
+        # retrocompatibility to accept tag as slug
```

Comment:
in the past we only support 1 tag
In a second time, we support several 1,2,3 only, but without slug
Now we support multi tag with slug.

Probably that since all this time, we could remove this old comment.
Double check the domain with id = None if you remove int parsing